app/components/ChatBox.js

```diff
@@ -58,7 +58,7 @@ function displayImage(text) {
 const MessageRender = ({ message }) => {
   const emojis = [];
   let match;
-  const regex1 = RegExp(/:[\-a-zA-Z_]+:/g);
+  const regex1 = RegExp(/:[\-a-zA-Z_|+0-9]+:/g);
```

Comment:
This is fine, but in this context (inside `[]`), `|` is matched literally, and anyway, you do not need an OR, because that's what the `[]` do anyway, permit any of the containing character.
So the following would be sufficient: `/:[\-a-zA-Z_+0-9]+:/g`

Reply:
ah yes